src/library/testing/CMakeLists.txt

```diff
@@ -25,7 +25,10 @@ create_test_sourcelist(_libf3dSDKTests libf3dSDKTests.cxx ${libf3dSDKTests_list}
 
 # add the executable
 add_executable(libf3dSDKTests ${_libf3dSDKTests})
-set_target_properties(libf3dSDKTests PROPERTIES RUNTIME_OUTPUT_DIRECTORY "${CMAKE_BINARY_DIR}/bin")
+set_target_properties(libf3dSDKTests PROPERTIES
+  RUNTIME_OUTPUT_DIRECTORY "${CMAKE_BINARY_DIR}/bin"
+  CXX_VISIBILITY_PRESET hidden
```

Comment:
Why do we need this ?

Reply:
without it, it produces linking warnings on macos